### ISSUE TITLE
Missing source reference for comments

### ISSUE THREAD

**Issue by aldhsu** [AleshaOleg/postcss-sass#72]
## Expected
Comments always have source

From postcss-scss parser

```
Comment {
        raws: { before: '\n', inline: true, left: ' ', right: ' ', text: 'a' },
        type: 'comment',
        parent:
         Root {
           raws: { semicolon: false, after: '' },
           type: 'root',
           nodes: [ [Circular], [Object] ],
           source: { input: [Object], start: [Object] } },
        source:
         { start: { line: 2, column: 1 },
           input: Input { css: '\n// a \n/* b */', id: '<input css 4>' },
           end: { line: 2, column: 5 } },
        text: 'a' }
```

## Actual
Comments do not have source

From snapshot:
https://github.com/AleshaOleg/postcss-sass/blob/0e6edadcb086a6c4217054b0a2abcd97cc9d5f98/__tests__/__snapshots__/parse.test.js.snap#L1147-L1156

## Potential Problem
http://api.postcss.org/global.html#source
> If you create a node manually (e.g., with postcss.decl()), that node will not have a source property and will be absent from the source map.

https://github.com/AleshaOleg/postcss-sass/blob/0e6edadcb086a6c4217054b0a2abcd97cc9d5f98/parser.es6#L343-L351

This is addressed in the other ndoe types but not in comments.

**Comment by Grawl** [comment]
https://github.com/stylelint/stylelint/issues/3244

**Comment by AleshaOleg** [comment]
Hey @aldhsu. Thanks for reporting, I'll try to fix this today.

**Comment by AleshaOleg** [comment]
@aldhsu, @Grawl fixed in `postcss-sass@0.3.5`
